api_tests/collections/test_views.py

```diff
@@ -1,174 +1,184 @@
+import pytest
 from urlparse import urlparse
 
-from nose.tools import *  # flake8: noqa
-
-from osf.models import AbstractNode, NodeLog
-from website.util.sanitize import strip_html
-from website.util import disconnected_from_listeners
-from website.project.signals import contributor_removed
 from api.base.settings.defaults import API_BASE
-from tests.base import ApiTestCase
+from framework.auth.core import Auth
 from osf_tests.factories import (
     CollectionFactory,
     NodeFactory,
     RegistrationFactory,
     ProjectFactory,
-    AuthUserFactory
+    AuthUserFactory,
 )
-from tests.utils import assert_logs
-from framework.auth.core import Auth
+from osf.models import AbstractNode, NodeLog
+from tests.utils import assert_logs, assert_items_equal
+from website.project.signals import contributor_removed
+from website.util.sanitize import strip_html
+from website.util import disconnected_from_listeners
 from website.views import find_bookmark_collection
 
 
-def node_url_for(node_id):
-    return '/{}nodes/{}/'.format(API_BASE, node_id)
+@pytest.fixture()
```

Comment:
I don't think this needs to be a fixture if it isn't using a model or complicated to compute